arch/arm/src/armv7-a/arm_syscall.c

```diff
@@ -579,12 +579,18 @@ uint32_t *arm_syscall(uint32_t *regs)
       addrenv_switch(NULL);
 #endif
 
+      cpu = this_cpu();
+      tcb = current_task(cpu);
+
+      /* Update scheduler parameters */
+
+      nxsched_suspend_scheduler(g_running_tasks[cpu]);
+      nxsched_resume_scheduler(tcb);
```

Comment:
It seems that this PR will cause the task CET(Case Execution Time) to be worst, right?

Reply:
no, both functions are only used to collect the statistics info, don't influence the behavior. without the fix, critmonitor report the wrong value.